alibi/explainers/similarity/grad.py

```diff
@@ -128,6 +129,14 @@ def __init__(self,
             task_name=task
         )
 
+        non_trainable_layers = self.backend.get_non_trainable(self.predictor)
+        if non_trainable_layers:
+            layers_msg = 'The following layers are not trainable: '
+            layers = ", ".join([f"'{layer}'" for layer in non_trainable_layers if layer is not None])
+            warning_msg = ("Some layers in the model are not trainable. These layer gradients will not be "
```

Comment:
`These layer gradients...` - presumably there are no gradients in the first place because the layers are not trainable? Perhaps need a small rephrase...